core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/PrimitiveActions.scala

```diff
@@ -575,7 +578,11 @@ protected[actions] trait PrimitiveActions {
       }
     }
 
-    activationStore.storeAfterCheck(activation, context)(transid, notifier = None)
+    activationStore.storeAfterCheck(activation, context)(
+      transid,
+      notifier = None,
+      blockingComposition,
+      disableStoreResultConfig)
```

Comment:
why is this configuration read in multiple places and passed to the underlying store? Can't it be ready just one?
The override is for debug mode and I think you can achieve this by having the following states:

- pass None: default pure config applies
- pass Some(t): t applies

The override uses Some(t) else None.

Reply:
I would use 1 param, indicating debug enabled. For the config, you can load it directly in `ActivationStore` and not pass it ever, I think. 

Also, I'm not sure why these params are implicit, since they are always explicitly set? If no reason, maybe just move them all to the first parameter set.

Reply:
+1

Reply:
`disableStoreResultConfig` was configured for `compositions` and `sequences` in order to be able to override it at testing. But given the fact we have moved the `storeDisable` logic in `ActivationStore` we could rely only on the `ActivationsApiTests`.
Will remove the parameter passing, and only keep the configuration in one place.